### ISSUE TITLE
Create appropriate domain forwarding from ESIP to new system, and test

### ISSUE THREAD

**Issue by graybeal** [ESIPFed/cor#17]
The principal goal here is to make sure an appropriate ESIP-level domain was where the site was hosted. As it stands, the site is at esipsw.org, which is a lovely domain name. Should ESIP decide they want it to be at, say, esip.org/cor,  at least a few installation changes would have to be made and tested.

**Comment by graybeal** [comment]
We (ESIP semantic and management teams) have decided to put the domain at cor.esipfed.org. I will contact the appropriate authorities to coordinate the change.

**Comment by graybeal** [comment]
Planned process is: (a) repoint the cor domain to 54.183.173.69, (b) update the path name(s) in the configuration file from esipsw.org to cor.esipfed.org, (c) restart services, (d) delete any existing ontologies (that will point to the wrong domain name) (e) edit the top header and welcome page to remove the red warnings, (f) reinstall the core ontology that drives the ontology type selection, and (g) redirect esipsw.org to cor.esipsw.org.

**Comment by graybeal** [comment]
Item (a) is done.

**Comment by graybeal** [comment]
Everything but item (g) is done, but links the use esipsw.org will still go to the correct IP address / end up at cor.esipfed.org, so I think it doesn't have to be relinked at this time.

The only hitch in the plan was that the software had to be recompiled before it all worked. And we had to remember our admin accounts on this machine.

We deleted the test ontologies, recreated the Authority ontology, and Carlos was able to rejigger the SML/sml ontology 'in place' using sed and similar magic.
